docs/pages/desktop-access/getting-started.mdx

```diff
@@ -25,78 +25,98 @@ This guide requires you to have:
   CS)](https://docs.microsoft.com/en-us/windows-server/networking/core-network-guide/cncg/server-certs/install-the-certification-authority)
   installed in the domain
 - Access to a Domain Controller
-- LDAP credentials for Active Directory (usually the same credentials you use
-  to log into the Domain Controller)
 - An existing Teleport cluster and user, version 8.0 or newer
-  - see [Teleport Getting Started](../getting-started.mdx) if you're new to Teleport
+  - See [Teleport Getting Started](../getting-started.mdx) if you're new to Teleport
 - A Linux server to run the Teleport Desktop Access service on
-  - you can reuse an existing server running any other Teleport instance
+  - You can reuse an existing server running any other Teleport instance
 
-## Step 1/3. Configure Teleport
+## Step 1/5: Create a User, Group, and Group Policy Object
+
+Teleport requires a service account to connect to your Active Directory domain. In theory any account with the required permissions can be used;
+we recommend creating a dedicated service account with restrictive permissions (described below) for maximal security.
+
+### Create User and Group
+
+From the start menu of your Domain Controller, search for “Active Directory Users and Computers” and open the corresponding program.
+
+Find your domain in the side menu, and right click the container you want the service account to live in (typically Users). From the context
+menu, select `New > User` and fill out the fields for your Teleport LDAP Service Account. For the "User logon name" field, you can use `svc-teleport`
+On the password screen, unselect  “User must change password at next login” and select “Password never expires”. Make a note of what values you put 
```

Comment:
Users will be concerned about old passwords that are insecure and unexpiring? Is there any way to make sure that these passwords are never used

Reply:
This password is used by teleport to talk to AD, but we disable interactive login and restrict the permissions so it can't be used to login to a host or do any actions outside of what teleport requires.